src/components/check/create/(sections)/OverviewSection.tsx

```
@@ -0,0 +1,21 @@
+import GeneralSection from '@/src/app/checks/create/GeneralSection'
+import QuestionsSection from '@/src/app/checks/create/QuestionsSection'
+import SettingsSection from '@/src/app/checks/create/SettingsSection'
+import Card from '@/src/components/Shared/Card'
+
+export function OverviewSection() {
+  return (
+    <>
+      <div className='mt-4 mb-8 flex flex-col gap-1'>
+        <span children='Preview changes' className='text-xl font-semibold' />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)